query/lib/src/core/expression/field.dart

```diff
@@ -27,6 +27,13 @@ class Field<ValType> {
   ///     find.where(age.eq(20));
   Cond<ValType> eq(ValType value) => Cond.eq<ValType>(this, value);
 
+  /// Returns an "is" condition, i.e. 'where var IS null'
+  ///
+  ///     FindStatement find = new FindStatement();
+  ///     Field<int> author = new Field<int>('age');
+  ///     find.where(age.eq(null));
```

Comment:
I guess the example is wrong here it use `eq`

Reply:
You can do everything as you wish, the main thing I need is the method I added =)

Reply:
No problem with the new method, but the doc should be correct or removed  ;)

Reply:
If you think it is better to do differently, you can do, I agree to this